examples/vesting-flow/index.html

```diff
@@ -150,16 +150,16 @@ <h2>Console</h2>
             .getTransactionsForContract(contractId)
             .then((result) =>
               Promise.all(
-                result.headers.map((txHeader) =>
+                result.transactions.map((transaction) =>
```

Comment:
This is a breaking change that I like, but it was not mentioned in the changelog